kie-server-parent/kie-server-api/src/main/java/org/kie/server/api/marshalling/json/JSONMarshaller.java

```diff
@@ -1160,4 +1171,8 @@ public void setClassLoader(ClassLoader classLoader) {
     public ClassLoader getClassLoader() {
         return classLoader;
     }
+
+    public void setXmlAnyElementsNames(boolean jsonHandleXmlAnyElementsNames) {
```

Comment:
is this method used only for testing purposes? if yes... maybe there's a better way to do so.... I personally like Anonymous classes for hack the tested class... but I also understand that it might not be the most popular design choice.